src/server/service/achievements.ts

```diff
@@ -45,7 +48,8 @@ const busterStreakAchievement: Achievement = {
     if (
       !playerHistory ||
       playerHistory.length < 2 ||
-      alreadyReceivedAchievementToday(player.userId, playerHistory, "101")
+      alreadyReceivedAchievementToday(player.userId, playerHistory, "101") ||
+      !player
```

Comment:
Probably worth inverting the order. so the conditional already returns false if there is no player. It will probably fail to call the next method since player is null.